src/messaging/tests/MessagingContext.h

```diff
@@ -92,16 +93,12 @@ class MessagingContext : public PlatformMemoryUser
     static constexpr System::Clock::Timeout kResponsiveIdleRetransTimeout   = System::Clock::Milliseconds32(10);
     static constexpr System::Clock::Timeout kResponsiveActiveRetransTimeout = System::Clock::Milliseconds32(10);
 
-    MessagingContext() :
-        mInitialized(false), mAliceAddress(Transport::PeerAddress::UDP(GetAddress(), CHIP_PORT + 1)),
-        mBobAddress(LoopbackTransport::LoopbackPeer(mAliceAddress))
-    {}
-    // TODO Replace VerifyOrDie with Pigweed assert after transition app/tests to Pigweed.
-    // TODO Currently src/app/icd/server/tests is using MessagingConetext as dependency.
-    ~MessagingContext() { VerifyOrDie(mInitialized == false); }
+    MessagingContext() : mpData(new MessagingContextData()) {}
+    // TODO Currently src/app/icd/server/tests is using MessagingContext as dependency.
```

Comment:
Please explain the todo: what is to be done? The TODO seems to only be a statement of "this is what is going on". Please rephrase it to describe what needs to be done and ideally reference a github issue describing the work (that someone can pick up and then later mark as fixed).

Reply:
The comment was from a previous developer.  I left it in the code since I didn't really understand the point of it, but that's probably a reason to remove it rather than leaving it.  Deleted.